### PR TITLE
C4k script

### PR DESCRIPTION
**Pull request by bamader**
## Background
Systemizes creating the C4K subset of ECE Reporter data to S3.

### CODE REVIEW

File: data-vis-workspace.code-workspace [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (kmagida110) [inline data-vis-workspace.code-workspace:1]: I assume we don't need this file....

File: src/build_db.py [hunk 2]
```diff
@@ -95,10 +95,13 @@ def get_july_2020_students():
 
 
 def get_smi_data():
-
     return pd.read_csv(SMI_AND_POVERTY_DATA)
 
 
+def get_c4k_subset_data():
```
Comment (kmagida110) [inline src/build_db.py:101]: Can you add c4k into clean_columns' name for clarity?

File: src/build_db.py [hunk 3]
```diff
@@ -119,7 +122,7 @@ def get_load_date(passed_args):
         try:
             loaded_date = datetime.strptime(input_date, '%Y-%m-%d')
         except ValueError:
-            raise Exception(f"{loaded_date} is an invalid input date, it must be in the form 'YYYY-MM-DD")
+            raise Exception(f"{input_date} is an invalid input date, it must be in the form 'YYYY-MM-DD")
```
Comment (kmagida110) [inline src/build_db.py:125]: Thanks!

**Review by kmagida110 — APPROVED** [review]
Looks good with a super minor change!

**Review by imcurtis — APPROVED** [review]

File: src/build_db.py (hunk 2 above)
Comment (bamader) [reply src/build_db.py:101]: Yeah good call

File: data-vis-workspace.code-workspace (hunk 1 above)
Comment (bamader) [reply data-vis-workspace.code-workspace:1]: Oh lol my VS code thing tryna sneak in

**Pull request merged**